samples/csharp_dotnetcore/13.basic-bot/README.md

```diff
@@ -14,111 +15,66 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
-  - Ensure "Copy to Output Directory" is set to "Copy always".
+4. Navigate to the sample folder (`botbuilder-samples/samples/csharp_dotnetcore/13.basic-bot`) and open `CoreBot.csproj` in Visual Studio
 
+5. Update your `appsettings.json` file
 
- 9. Update your `appsettings.json` with your .bot file path and .bot file secret.
-- Update the `botFilePath` with the name of your generated bot configuration file.
-- Update the `botFileSecret` with the secret that the msbot tool generated from step 5 above.
+    5.1. Complete the properties's values with the mentioned keys
 
-```bash
-{
-    "botFilePath": "basic-bot.bot",
-    "botFileSecret": "<YOUR BOT SECRET>"
-}
-```
+    - `MicrosoftAppId` with your generated Microsoft Application Id
+    - `MicrosoftAppPassword` with the Application Secret generated
+    - `LuisAppId` with your generated LUIS Application Id
+    - `LuisAPIKey` with the LUIS API Key generated
+    - `LuisAPIHostName` with the endpoint's name
 
-## [Alternate to CLI] Set up LUIS via Portal
-- Navigate to [LUIS portal](https://www.luis.ai).
-- Click the `Sign in` button.
-- Click on the `Choose File` and select [basic-bot.json](basic-bot.json) from the `botbuilder-samples/csharp_dotnetcore/13.basic-bot/CognitiveModels` folder.
-- Update [basic-bot.bot](basic-bot.bot) file with your AppId, SubscriptionKey, Region and Version. 
-    You can find this information under "Manage" tab for your LUIS application at [LUIS portal](https://www.luis.ai).
-    - The `AppID` can be found in "Application Information"
-    - The `SubscriptionKey` can be found in "Keys and Endpoints", under the `Key 1` column
-    - The `region` can be found in "Keys and Endpoints", under the `Region` column
-- Update [basic-bot.bot](basic-bot.bot) file with your Authoring Key.  
-    You can find this under your user settings at [luis.ai](https://www.luis.ai).  Click on your name in the upper right hand corner of the portal, and click on the "Settings" menu option.
-    NOTE: Once you publish your app on LUIS portal for the first time, it takes some time for the endpoint to become available, about 5 minutes of wait should be sufficient.
-- Update [basic-bot.bot](basic-bot.bot) file to ensure the `Id` property on the `luis` service type is set to `basic-bot-LUIS`.
+    ```bash
+    {
+        "MicrosoftAppId": "<YOUR MICROSOFT APP ID>",
+        "MicrosoftAppPassword": "<YOUR MICROSOFT APP PASSWORD>"
+        "LuisAppId": "<YOUR LUIS APP ID>",
+        "LuisAPIKey": "<YOUR LUIS API KEY>"
+        "LuisAPIHostName": "<YOUR LUIS HOST NAME>"
+    }
+    ```
 # Running Locally
 
 ## Visual Studio
-- Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/13.basic-bot`) and open `BasicBot.csproj` in Visual Studio.
-- Run the project (press `F5` key)
+1. Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/13.basic-bot`) and open `CoreBot.csproj` in Visual Studio
+3. Run the project (press `F5` key)

```

Comment:
I think this'd be a 2. instead of 3., right?

Reply:
Done